### DISCUSSION TITLE
Recommended Installation V1 versus V2

### DISCUSSION THREAD

**Discussion by Wind4Greg** [parcel-bundler/parcel#6953] [Q&A] [answered]
Hi, very much like parcel and use it when teaching my web development classes. In Parcel version 1 it was recommended in the [Getting Started](https://parceljs.org/getting_started.html) to do a global install, e.g., `npm install -g parcel-bundler`. In the [V2 documentation](https://v2.parceljs.org/getting-started/webapp/) this has changed to a local install, i.e., `npm install --save-dev parcel`.

Is there a reason for this change in recommendation? In a classroom situation we tend to do a lot of small projects and a global install reduces installation/setup overhead, but if there are "gotchas" or other potential issues. I don't want to recommend a global install with V2 to my students.

Thanks a bunch for any insights

Greg B.

**Comment by mischnic** [comment, marked as answer]
I would always install Parcel locally in the project. Otherwise, upgrading the global Parcel version could break some of your projects without you even realizing it immediately.
